sky/shell/gpu/direct/ganesh_canvas.cc

```diff
@@ -32,6 +33,22 @@ bool GaneshCanvas::SetupGrGLInterface() {
       kOpenGL_GrBackend,
       reinterpret_cast<GrBackendContext>(GrGLCreateNativeInterface())));
 
+#ifdef NDEBUG
```

Comment:
I'd flip this and put ifndef NDEBUG first, just because that's the more common pattern.